Xamarin.Forms.Platform.iOS/Material/MaterialButtonRenderer.cs

```diff
@@ -0,0 +1,336 @@
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using Foundation;
+using UIKit;
+using Xamarin.Forms.PlatformConfiguration.iOSSpecific;
+using SizeF = CoreGraphics.CGSize;
+using MButton = MaterialComponents.Button;
+using Xamarin.Forms;
+
+[assembly: ExportRenderer(typeof(Xamarin.Forms.Button), typeof(Xamarin.Forms.Platform.iOS.Material.MaterialButtonRenderer), new[] { typeof(VisualRendererMarker.Material) })]
+namespace Xamarin.Forms.Platform.iOS.Material
+{
+	public class MaterialButtonRenderer : ViewRenderer<Button, MButton>
```

Comment:
I'm curious why, as `MButton` inherits from `UIButton`, you didn't make the MaterialButtonRenderer inherits from ButtonRenderer ?
I get it's easier to start from a new clean slate, or at least more fun, but I'm afraid that doing a whole new set of renderers will make resurface bugs we solved in the past. It's also 2 codepaths to maintain in the future, and this isn't a situation like AppCompat where there's a plan to deprecate one set...

Reply:
This will be refactored to inherit. Probably today or tomorrow. It just became too difficult while working on this to differentiate API changes. I built the material control up in a predictable way and now that I have a strong idea of where the two need to have split logic I can just base class the ButtonRenderer. On the android side as well I'm planning on moving out similar logic to static classes